### PR TITLE
Featured/closed

### PR DESCRIPTION
**Pull request by hasft**

### CODE REVIEW

File: .changeset/brown-foxes-perform.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ufjt-poc/bot": patch
```
Comment (buljagung) [inline .changeset/brown-foxes-perform.md:2]: loh

Comment (hasft) [reply .changeset/brown-foxes-perform.md:2]: ok

Comment (buljagung) [reply .changeset/brown-foxes-perform.md:2]: kakakak

Comment (hasft) [reply .changeset/brown-foxes-perform.md:2]: dampu

Comment (hasft) [reply .changeset/brown-foxes-perform.md:2]: ok deh

Comment (buljagung) [reply .changeset/brown-foxes-perform.md:2]: lala

Comment (buljagung) [reply .changeset/brown-foxes-perform.md:2]: dddd

Comment (buljagung) [reply .changeset/brown-foxes-perform.md:2]: okokok

**Pull request closed without merging**